src/CapsLimit/CapsLimit.php

```diff
@@ -111,9 +111,10 @@ public function onChat(PlayerChatEvent $event){
                 $player->kick("You have been kicked for overused caps!");
             }
             if($count > $this->getMaxCaps() and $this->getConfig()->getNested("broadcast.enable") === true){
+                $subject = $this->getConfig()->getNested("broadcast.message");
+                $subject = str_replace("{PLAYER}", $player->getName(), $subject);
                 foreach($this->getServer()->getOnlinePlayers() as $p){
-                    $subject = $this->getConfig()->getNested("broadcast.message");
-                    $p->sendMessage($this->getPrefix().TextFormat::RED.str_replace("{PLAYER}", $player->getName(), $subject));
+                    $p->sendMessage($this->getPrefix().TextFormat::RED.$subject);
```

Comment:
why not just do the whole thing and move the prefix and colour outside the loop too?